tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/HotChocolateTests.cs

```diff
@@ -0,0 +1,261 @@
+                {
+                    if (args.Data != null)
+                    {
+                        Output.WriteLine($"[webserver][stderr] {args.Data}");
+                    }
+                };
+                process.BeginErrorReadLine();
+
+                wh.WaitOne(15_000);
+                if (!aspNetCorePort.HasValue)
+                {
+                    throw new Exception("Unable to determine port application is listening on");
+                }
+
+                Output.WriteLine($"The ASP.NET Core server is ready on port {aspNetCorePort}");
+
+                var expectedSpans = SubmitRequests(aspNetCorePort.Value);
+
+                if (!process.HasExited)
+                {
+                    // Try shutting down gracefully
+                    var shutdownRequest = new RequestInfo() { HttpMethod = "GET", Url = "/shutdown" };
+                    SubmitRequest(aspNetCorePort.Value, shutdownRequest);
+
+                    if (!process.WaitForExit(5000))
+                    {
+                        Output.WriteLine("The process didn't exit in time. Taking proc dump and killing it.");
+                        var memoryDumpCaptured = await TakeMemoryDump(process);
+
+                        process.Kill();
+
+                        if (!memoryDumpCaptured)
+                        {
+                            // if we don't have a memory dump, there's no point continuing.
+                            // We know the test will likely fail (telemetry not sent) but it's
+                            // not useful to know that, as we don't have a memory dump
+                            throw new SkipException("The process didn't exit in time but memory dump couldn't be captured");
+                        }
+                    }
+                }
+
+                var spans = agent.WaitForSpans(expectedSpans);
+
+                var settings = VerifyHelper.GetSpanVerifierSettings();
+
+                // hacky scrubber for the fact that version 4.1.0+ switched to using " in error message in one place
+                // where every other version uses '
+                settings.AddSimpleScrubber("Did you mean \"appearsIn\"", "Did you mean 'appearsIn'");
+                // Graphql 5 has different error message for missing subscription
+                settings.AddSimpleScrubber("Could not resolve source stream for field", "Error trying to resolve field");
+
+                // Overriding the type name here as we have multiple test classes in the file
+                // Ensures that we get nice file nesting in Solution Explorer
+                var fxSuffix = EnvironmentHelper.IsCoreClr() ? string.Empty : ".netfx";
+                await VerifyHelper.VerifySpans(spans, settings)
+                                  .UseFileName($"{_testName}.SubmitsTraces{fxSuffix}")
+                                  .DisableRequireUniquePrefix(); // all package versions should be the same
```

Comment:
This is left over copy-pasta from the GraphQL integration. I think you can replace it with the following (Note that you would need to rename the snapshot file too)
```suggestion
                await VerifyHelper.VerifySpans(spans, settings)
                                  .UseFileName("HotChocolateTests.SubmitsTraces")
                                  .DisableRequireUniquePrefix(); // all package versions should be the same
```